tools/test-api

```diff
@@ -58,7 +60,7 @@ with test_server_running(force=options.force, external_host='zulipdev.com:9981')
     email = 'guest@zulip.com'  # guest is not an admin
     guest_user = do_create_user('guest@zulip.com', 'secret',
                                 get_realm('zulip'), 'Mr. Guest', 'guest')
-    api_key = guest_user.api_key
+    api_key = get_api_key(guest_user)
```

Comment:
The refactor in this file (and many others) could easily be done in a preparatory commit, that introduces `get_api_key(user_profile)` as just `return user_profile.api_key`.  And then the later commit would just need to change `get_api_key`.